rotkehlchen/exchanges/binance.py

```diff
@@ -202,6 +217,7 @@ def __init__(
         self.msg_aggregator = msg_aggregator
         self.offset_ms = 0
         self.selected_pairs = binance_selected_trade_pairs
+        self.history_events_db = DBHistoryEvents(self.db)
```

Comment:
you don't need to do this. This is just a wrapper that is initialized ony when needed. No need to create it and keep it in memory for the entire duration of the lifetime of the object